src/printer.js

```diff
@@ -2106,11 +2122,29 @@ function printArgumentsList(path, options, print) {
     const shouldBreak = shouldGroupFirst
       ? printed.slice(1).some(willBreak)
       : printed.slice(0, -1).some(willBreak);
+
+    let printedLastArgExpanded;
+    if (shouldGroupFirst) {
+      printedLastArgExpanded = printed;
+    } else {
+      // We want to print the last argument with a special flag
+      let i = 0;
+      path.each(function(argPath) {
```

Comment:
`path.call` below should instead be `argPath.call`. It happens to be working because `path` is being mutated during the traversal. Here I want to iterate on all the arguments and reprint the last one with the special flag.